pkg/config/token_signing_config.go

```diff
@@ -25,7 +27,22 @@ type TokenSigningConfig struct {
 	// configuration.
 	Keys keys.Config `env:",prefix=TOKEN_"`
 
-	TokenSigningKey   string `env:"TOKEN_SIGNING_KEY, required"`
-	TokenSigningKeyID string `env:"TOKEN_SIGNING_KEY_ID, default=v1"`
-	TokenIssuer       string `env:"TOKEN_ISSUER, default=diagnosis-verification-example"`
+	TokenSigningKey   []string `env:"TOKEN_SIGNING_KEY, required"`
```

Comment:
Probably doc this:

- Slice of keys, first key is active, other keys are still accepted